### ISSUE TITLE
question about training steps in the paper

### ISSUE THREAD

**Issue by shuxiusuxiu** [volcengine/verl#1643]
I've been studying this framework these past few days.
I am just confused about the training steps.

For example, in the config(recipe/dapo/run_dapo_qwen2.5_32b.sh), `train_prompt_bsz=512` and `trainer.total_epochs=1`, the DAPO-math-17k has approximately 17000 samples, so the total steps should be `17000 / 512`?

Why does the image in the paper show thousands of steps?

**Comment by eric-haibin-lin** [comment]
@tongyx361

**Comment by tongyx361** [comment]
Thanks for your question.
The training steps in the paper are gradient updating steps that align with the R1 setup.

**Comment by xuwenxinedu** [comment]
> Thanks for your question. The training steps in the paper are gradient updating steps that align with the R1 setup.

So, the answer is that a ppo_mini_batch_size is a step ?
